src/core/a-entity.js

```diff
@@ -423,15 +425,15 @@ var proto = Object.create(ANode.prototype, {
   },
 
   removeComponent: {
-    value: function (name) {
+    value: function (name, force) {
```

Comment:
`canvas` and `look-controls` are default components on `<a-scene>` that need resetting during `reload`. Removing them and re-adding them seemed the best way to simulate the initial load and the closest to what happens to components on other elements.